dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -56,5 +56,7 @@ RUN apt-get purge -y           \
 
 COPY ["docker_init.sh", "lm-sensors.sh", "/usr/bin/"]
 COPY ["docker-pmon.supervisord.conf.j2", "start.sh.j2", "/usr/share/sonic/templates/"]
+RUN ls -l
```

Comment:
What is the purpose of this?

Reply:
Just add to the build log the list of files copied in the next line

Reply:
But this command is not run in the ssd_tools/ directory. It doesn't seem like it will list the files you want to list.

Reply:
right you are... looks like it lists the "target"
removed from the script

Reply:
right you are... looks like it lists the "target"
removed from the script